tests/resource_tests/generator_tests/test_participant_enrollment.py

```diff
@@ -237,7 +256,7 @@ def test_previous_ehr_unsure_with_dv_yes(self):
 
         # Get Core status through EHR consents
         self._submit_ehrconsent(self.participant_id,
-                                response_code=CONSENT_PERMISSION_NOT_SURE,
+                                response_code=CONSENT_PERMISSION_YES_CODE,
```

Comment:
Somewhere along the way I got the idea that an EHR submission of UNSURE meant that a DV_EHR of YES could allow for CORE status. Looking through the code now I can't remember what gave me that idea. Thanks for fixing up this test. I'll investigate further and make sure no other changes are needed and if anyone is affected by my mistake.